lib/seed/facility_seeder.rb

```diff
@@ -62,19 +62,22 @@ def call
       puts "Creating #{number_of_facility_groups} FacilityGroups..."
 
       facility_groups = number_of_facility_groups.times.map {
-        FactoryBot.build(:facility_group, organization_id: organization.id, state: nil)
+        state_region = FactoryBot.create(:region, :state, name: Seed::FakeNames.instance.state, reparent_to: organization.region)
```

Comment:
This will create a `State` per `FacilityGroup`, is that ok?

Reply:
Hm, we'd rather have a bunch of districts in facility groups, which more closely matches prod.  I'll take a shot at it.

Reply:
Check out https://github.com/simpledotorg/simple-server/pull/1571, I'm also going to kick off a large benchmark run here to make sure things stay fast.

Reply:
Closing this PR.